chapters/08-documentation.Rmd

```diff
@@ -10,4 +10,24 @@
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+1. **Creating a vignette** 
+    * Create a template vignette for your package by running `use_vignette(FIXME, title = FIXME)`.
+    * Modify the vignette to include:
+	- a high-level paragraph description of your package and its purpose.
+	- an example of how to use your package, including code.
+    * Use `knitr` to preview your vignette.
+
+2. **Documenting functions**
+    * Inspect the `roxygen` comments in your functions (lines beginning with `#'`). 
+    Add or edit at least one example for each function, and make sure the `param`, `return`, and `export` fields match the function behavior.
+    * Run `document()` to convert your function documentation to `.Rd` files. 
+
+3. **Creating a README**
```

Comment:
Should the README and vignette be pretty much the same? What is the procedure for making this happen?

Reply:
I think no, since in the next chapter when we cover pkgdown, these end up in different places - the README on the landing page, but vignettes via the top nav bar.

Reply:
Minor thing, I'd suggest creating the README be done before the vignette.